### ISSUE TITLE
Dead pixel appears with skype for Business web app 

### ISSUE THREAD

**Issue by alanpadillachua** [OfficeDev/skype-docs#371]
<img width="148" alt="screen shot 2017-10-10 at 11 10 01 am" src="https://user-images.githubusercontent.com/22553942/31397418-97ed7c98-adab-11e7-98bc-ca148a1665b0.png">

A dead pixel appears every time the web app is open. As soon as I exit and quit the app the dead pixel goes away.

**Comment by andrewjrangel** [comment]
I have seen this as well on Safari

**Comment by julianiaccopucci** [comment]
Yes, I can see that too. It happens on the screen you initialise the program on. The pixel extends to the screen and it stays still. It's gone after the program is closed

**Comment by chesterlaw** [comment]
It's still there.

**Comment by mslonina** [comment]
It happens every time I use the app (even on external monitor) - MacOS Mojave, MacbookPro2017TB, RadeonPro 560 - no other app is causing such an issue, also, the Energy tab in Activity monitor shows, that skype does not use high performance gpu

**Comment by pguyot** [comment]
This is surprising and scary. Yet if you can screenshot it, it is not a dead pixel :)

It seems to be the minimization point of the Web App Plug-in alert asking the authorization to access audio, video and screen. If you check the box "Always allow the plug-in for this domain", the "dead pixel" will not appear in subsequent calls with the same domain as the alert will not be shown.

<img width="437" alt="pluginalert" src="https://user-images.githubusercontent.com/168407/49747411-ac933f00-fca3-11e8-90a1-690d32cf8dc7.png">
